README.md

```diff
@@ -1,34 +1,43 @@
+Welcome to our Github repository! This app is built with Next.js, Tailwind.css, pnpm, Prisma, and Supabase.
 
-## Getting Started
+# Running the app
 
-First, run the development server:
+## Testing database
 
-```bash
-npm run dev
-# or
-pnpm run dev
-```
+To run the app with testing database, follow these steps:
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
+### Prerequisities
 
-You can start editing the page by modifying `pages/index.tsx`. The page auto-updates as you edit the file.
+-   [Node.js](https://nodejs.org/en/)
+-   [pnpm](https://pnpm.io/)
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.ts`.
+### Steps
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+1. Clone this repository to your local machine.
+1. Navigate to the root directory of the project and run `pnpm install` to install the dependencies.
+1. Set the environment variables in a `.env` file in the root directory. You can find a sample `.env.testing` file in the root directory.
+1. Run `pnpm dev` to start the development server.
+1. Open your browser and go to http://localhost:3000 to view the app.
 
-## Learn More
+## Local database
 
-To learn more about Next.js, take a look at the following resources:
+To run the app with local database, follow these steps:
 
--   [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
--   [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+### Prerequisities
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+-   [Node.js](https://nodejs.org/en/)
+-   [pnpm](https://pnpm.io/)
+-   [Docker](https://www.docker.com/)
+-   Make sure your Docker is running
 
-## Deploy on Vercel
+### Steps
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
-
-Check out our [Next.js deployment documentation](https://nextjs.org/docs/deployment) for more details.
+1. Clone this repository to your local machine.
+1. Navigate to the root directory of the project and run `pnpm install` to install the dependencies.
+1. Set the environment variables in a `.env` file in the root directory. You can find a sample `.env.testing` file in the root directory.
+1. Run `pnpm supabase start`.
+1. Copy `DB URL` into `.env` file to the `DATABASE_URL` and remove the #.
+1. Run `pnpm prisma migrate deploy`.
+1. Run `pnpm prisma db seed`.
```

Comment:
It fails with error: @prisma/client did not initialize yet.

Reply:
Thanks, I will try to replicate this issue and test with other users. Maybe we can take a look? @Mmykron